spec/controllers/media_controller_spec.rb

```diff
@@ -106,6 +115,17 @@
         expect(body['status']).to eq 'success'
         expect(body['token']).to eq 'sekret-token'
       end
+
+      it 'still returns info about applicable access restrictions' do
```

Comment:
I'm not sure I understand "still" here?  Can we remove that?

Reply:
ah, yeah, was thinking relative to the old behavior where access restriction info was no longer returned once the user is successfully logged in.  but i can see how that's confusing, removed.